command/sub8_missions/nodes/move_command

```diff
@@ -37,8 +47,49 @@ def main(args):
         fprint("Waiting for odom...")
         yield sub.tx_pose()
         fprint("Odom found!", msg_color='green')
-
+        rospack = rospkg.RosPack()
+        config_file = os.path.join(rospack.get_path('sub8_missions'), 'nodes', 'tp.yaml')
+        f = yaml.load(open(config_file, 'r'))
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
+        if args.gteleport:
```

Comment:
I couldn't get this to run. If only ```subc -gt "buoy"``` is entered, the code can never get to this line. How do you use it?